File: includes/undisposable_clients/php/xmlrpc/undisposable.inc.php
<?php
require_once 'XML/RPC.php';

function undorg_isDisposableEmail($input) {
    $params = array(new XML_RPC_Value($input, 'string'));
    $msg = new XML_RPC_Message('isDisposableEmail', $params);

    $cli = new XML_RPC_Client('/services/xmlrpc/isDisposableEmail/index.php', 'www.undisposable.org');
    $resp = $cli->send($msg);

    if (!$resp) {
	echo 'Communication error: ' . $cli->errstr;
        exit;
    }

    if (!$resp->faultCode()) {
	$val = $resp->value();
        return $val->getVal();
    } else {
	/*
         * Display problems that have been gracefully cought and
	 * reported by the xmlrpc.php script.
         */
	// echo 'Fault Code: ' . $resp->faultCode() . "\n";
        // echo 'Fault Reason: ' . $resp->faultString() . "\n";
        return false;
    }
}


function undorg_isDisposableHost($input) {
    $params = array(new XML_RPC_Value($input, 'string'));
    $msg = new XML_RPC_Message('isDisposableHost', $params);

    $cli = new XML_RPC_Client('/services/xmlrpc/isDisposableHost/index.php', 'www.undisposable.org');
    $resp = $cli->send($msg);

    if (!$resp) {
	echo 'Communication error: ' . $cli->errstr;
        exit;
    }

    if (!$resp->faultCode()) {
	$val = $resp->value();
        return $val->getVal();
    } else {
	/*
         * Display problems that have been gracefully cought and
	 * reported by the xmlrpc.php script.
         */
	// echo 'Fault Code: ' . $resp->faultCode() . "\n";
        // echo 'Fault Reason: ' . $resp->faultString() . "\n";
        return false;
    }
}


?>

File: includes/undisposable_clients/java/xmlrpc/undisposable.inc.java
import java.util.Vector;
import helma.xmlrpc.*;

public class Undisposable {

    // The location of our server.
    private static String server_url =
        "http://www.undisposable.org/services/xmlrpc/";

    public static boolean isDisposableEmail (String email) {
        try {

            server_url += "isDisposableEmail/index.php"

            // Create an object to represent our server.
            XmlRpcClient server = new XmlRpcClient(server_url);

            // Build our parameter list.
            Vector params = new Vector();
            params.addElement(email);

            // Call the server, and get our result.
            boolean result =
                (boolean) server.execute("isDisposableEmail", params);

            return result;

        } catch (XmlRpcException exception) {
            /* System.err.println("JavaClient: XML-RPC Fault #" +
                               Integer.toString(exception.code) + ": " +
                               exception.toString());
            */
            return false;
        } catch (Exception exception) {
            // System.err.println("JavaClient: " + exception.toString());
            return false;
        }
    }
}
